hepdata/modules/records/utils/records_update_utils.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+import requests
+import math
+
+from hepdata.modules.records.utils.doi_minter import generate_dois_for_submission
+from hepdata.modules.submission.api import get_latest_hepsubmission
+from hepdata.modules.records.utils.workflow import update_record
+from hepdata.modules.inspire_api.views import get_inspire_record_information
+from hepdata.ext.elasticsearch.api import index_record_ids
+from hepdata.modules.email.api import notify_publication_update
+
+
+def update_record_info(inspire_id, recid='', send_email=False):
+    inspire_id = inspire_id.replace("ins", "")
+
+    if recid == '':
+        hep_submission = get_latest_hepsubmission(inspire_id=inspire_id)
+        if hep_submission is None:
+            print("Failed to retrieve hep submission for {0}".format(inspire_id))
+            return
+        recid = hep_submission.publication_recid
+
+    updated_record_information, status = get_inspire_record_information(inspire_id)
+
+    if status == 'success':
+        record_information = update_record(recid, updated_record_information)
+    else:
+        print("Failed to retrieve publication information for {0}".format(inspire_id))
+        return
+
+    if recid == '' and hep_submission.overall_status == 'finished':
+        index_record_ids([record_information["recid"]])
+        generate_dois_for_submission.delay(inspire_id=inspire_id)  # update metadata stored in DataCite
+        if send_email:
+            notify_publication_update(hep_submission, record_information)   # send email to all participants
+
+
+def get_all_updated_records_since_date(date, verbose=False):
+    """
+    Returns all inspire ids of records updated since the specified date (YYYY-MM-DD).
+    Alternatively, date can be a number, denoting days since today (-1 = yesterday).
+    Only records updated on that specific date will then be returned.
+    """
+
+    date_is_delta_from_today = (type(date) is int or type(date) is str and date.isdigit())
+
+    if verbose:
+        if date_is_delta_from_today:
+            print("Obtaining inspire ids of records updated {} days ago.".format(abs(int(date))))
+        else:
+            print("Obtaining inspire ids of records updated since {}.".format(date))
+
+    if date_is_delta_from_today:
+        specified_time = datetime.datetime.today() + datetime.timedelta(days=-int(date))
+    else:
+        specified_time = datetime.datetime.strptime(date, "%Y-%m-%d")
+
+    page = 1
+    counter = 0
+    ids = []
+
+    while page < 1000:
```

Comment:
Can you calculate the number of pages required from `response.json()['hits']['total']`?

Reply:
Done!